apps/sendsignal.go

```diff
@@ -0,0 +1,94 @@
+package apps
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/go-scalingo/v6"
+)
+
+func isContainerInList(containers []scalingo.Container, containerToCheck scalingo.Container) bool {
+	for _, container := range containers {
+		if container.ID == containerToCheck.ID {
+			return true
+		}
+	}
+	return false
+}
+
+func keepUniqueContainersWithPrefixes(containers []scalingo.Container, prefixes []string) []scalingo.Container {
+	containersToStop := make([]scalingo.Container, 0)
+
+	for _, prefix := range prefixes {
+		hasMatched := false
+		for _, container := range containers {
+			if strings.HasPrefix(container.Label, prefix) && !isContainerInList(containersToStop, container) {
+				containersToStop = append(containersToStop, container)
+				hasMatched = true
+			}
+		}
+		if !hasMatched {
+			fmt.Printf("-----X The prefix '%v' did not match any container\n", prefix)
+		}
+	}
+
+	return containersToStop
+}
+
+func keepUniqueContainersWithNames(containers []scalingo.Container, names []string) []scalingo.Container {
+	containersToStop := make([]scalingo.Container, 0)
```

Comment:
best practice: we rarely use `make`
naming: you don't stop anything

```suggestion
	containersToKill := []scalingo.Container{}
```